src/integrationTest/kotlin/io/provenance/p8e/plugin/ContractPluginIntegrationTest.kt

```diff
@@ -72,9 +72,9 @@ class ContractPluginIntegrationTest : WordSpec() {
                 cleanResult.task(":contracts:clean") shouldNot haveOutcome(TaskOutcome.FAILED)
                 p8eCleanResult.task(":p8eClean") should haveOutcome(TaskOutcome.SUCCESS)
 
-                FileUtils.listFiles(File(contractPath), WildcardFileFilter("ContractHash*.kt"), WildcardFileFilter("."))
+                FileUtils.listFiles(File(contractPath), WildcardFileFilter.builder().setWildcards("ContractHash*.kt").get(), WildcardFileFilter.builder().setWildcards(".").get())
```

Comment:
random deprecated stuff, good thing a builder for a simple value is _so much_ more concise than a single constructor